common/src/androidTest/kotlin/JSONArrayBugInstrumentedTest.kt

```diff
@@ -0,0 +1,33 @@
+import android.os.Build
+import com.revenuecat.purchases.common.networking.MapConverter
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+
+//@org.junit.runner.RunWith(androidx.test.ext.junit.runners.AndroidJUnit4::class)
```

Comment:
Why is this commented out? We might need to setup the test runner in the build.gradle...